### ISSUE TITLE
Inverse Median Frequency Weights functioning?

### ISSUE THREAD

**Issue by Oscared** [nshaud/DeepHyperX#12]
Hi!

I was using this framework (thank you), but I cannot understand if the inverse median frequency weights are called at all? For me it seems they are created after constructing the loss function and thus never passed anywhere. Is this assumption wrong on my part? Where do they come into play? I assumed the weights would have been passed to the get_model function through the hyperparameters, then used to construct the loss function and not be overwritten by the new initialization of the weight vector. Thank you!

**Comment by Ayush-iitkgp** [comment]
I can confirm the same. My data has a huge class imbalance and the model performances are very poor.
